src/app/code/community/Shopgate/Cloudapi/Model/Api2/Customers/Addresses/Rest/Customer/V2.php

```diff
@@ -64,4 +69,18 @@ protected function _loadCustomerById($id)
 
         return $customer;
     }
+
+    /**
+     * @param array $data
+     *
+     * @throws Mage_Api2_Exception
+     */
+    protected function _multiDelete(array $data)
```

Comment:
any reason it is in this class? You see, logic of this class only happens for the customer caller. Hence why there are checks like `$this->getApiUser()->getUserId() !== $customer->getId()` in the other functions. They make sure no malicious user is trying to create/retrieve/delete someone else's stuff. The multi-delete function may also be needed for the admin user call when we have an admin panel or something, so it does not really belong here.